src/libsync/propagateuploadng.cpp

```diff
@@ -47,12 +47,15 @@ QUrl PropagateUploadFileNG::chunkUrl(qint64 chunk)
     return Utility::concatUrlPath(propagator()->account()->url(), path);
 }
 
-void PropagateUploadFileNG::slotZsyncSeedFinished(void* zs) {
-    _zs = (struct zsync_state *)zs;
+void PropagateUploadFileNG::slotZsyncSeedFinished(void* zs)
+{
+    zsync_unique_ptr<struct zsync_state> _zs((struct zsync_state*)zs, [](struct zsync_state* zs) {
```

Comment:
Here you removed the member, but the underscore is confusing now. Could you rename this and `_zbyterange`, etc?

Reply:
Will do.